### DISCUSSION TITLE
Way to connect CombineTest to GSPro?

### DISCUSSION THREAD

**Discussion by n00bco** [christianrehn/GCQuadCombineTest#7] [Q&A] [answered]
Hi — GSPro leverages the same shot data from Foresight to facilitate its sim software. Insofar as FSX Play is unstable, crashing every other time it’s loaded in my case, I would like to request some basic support to redirect to a relevant data source attached to GSPro. Desired experience is GSPro Practice Range instead of FSX Play on Display1, CombineTest on Display2. Would this be possible? The GSPro discord server’s 18.2k users would really appreciate this, I’m sure. Cheers

**Comment by christianrehn** [comment, marked as answer]
Hi, I do not have GSPro and there is no demo version available. Do you know if GSPro can export shot data after each shot in cvs, json or some other format like FSX 2020 (according to the documentation https://gsprogolf.com/GSProConnectV1.html the GSPro Open Connect API does only consume but not export shot data)? If this is possible, could you please send me an example shot data export. With this example file I could adjust the CombineTest shot data import for GSPro.

**Reply by n00bco** [reply]
Will look into it / ask around. There is an Export Data function in GSPro (within the Range, for performance analysis) but it's a manual export only, to my knowledge -- will confirm. Thank you
